### ISSUE TITLE
trigger event

### ISSUE THREAD

**Issue by tarjoadi** [dbolya/yolact#309]
Hi,

Is there a way to trigger an event if an certain object is detected (or to return the coordinates) ?

Thanks

**Comment by bayraktare** [comment]
Hi, 
You mean bounding box coordinates?
If so, you can get them from eval.py https://github.com/dbolya/yolact/blob/db81124874817895db69f2dc443f5c24e0e3f491/eval.py#L160
in addition to the classes and confidence rates.

**Comment by dbolya** [comment]
Also #256 might be of use.

**Comment by Marwen-Bhj** [comment]
> Hi,
> You mean bounding box coordinates?
> If so, you can get them from eval.py
> 
> https://github.com/dbolya/yolact/blob/db81124874817895db69f2dc443f5c24e0e3f491/eval.py#L160
> 
> 
> in addition to the classes and confidence rates.

x what about the masks themselves I need them in order to extract the detected objects without a background

**Comment by areebsyed** [comment]
@Marwen-Bhj did you get any solution for masks? I also want to do that

**Comment by Marwen-Bhj** [comment]
@areebsyed  yes you can refer to this https://github.com/dbolya/yolact/issues/286
